src/main/templates/languages.hbs

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  {{>header}}
+  <!-- Main content starts - do not edit above this line -->
+    <table class="table table-striped table-small table-hover">
```

Comment:
Stripping does not seem to work when searching.

Reply:
Ah yea, forgot to remove this when I fixed the search. Will update in a few minutes.